src/spatialref.jl

```diff
@@ -1,3 +1,85 @@
+"""
+Import format and run function `f` on the result.
+
+The name could be improved?
```

Comment:
I think [`newspatialref`](https://github.com/yeesian/ArchGDAL.jl/blob/fe9ee2a312eafac9d00f9f43100b0dd7f5a005e6/src/spatialref.jl#L6-L7) could work here. The docstring will then be something analogous like `"Construct a Spatial Reference System from a GeoFormat."` What do you think?

Reply:
This didn't turn out to be workable